src/server/index.ts

```diff
@@ -122,9 +122,11 @@ app.use(async (req: Request, res: Response) => {
     // await getDataFromTree(el) fails if any GraphQl request fails.
     // Try to generate html withour apollo state
     // TODO: Downside is that page is re-rendered on client which causes blinking
+    console.error(e);
```

Comment:
Hmm, would it be beneficial if we added some context to the error? Some string that says where things went wrong? When this error happens?

I mean this might be enough, just wondering

Reply:
sure, I can add that :+1: